### PR TITLE
Workflow removed for docs generation

### LINKED ISSUE #2426: Feature Request: Remove the workflow which generates docs
**Is your feature request related to a problem? Please describe.**
Remove because docs are getting fetched from docs repo, workflow is present there.

**Describe the solution you'd like**
 this is redundant code, need to be removed

**Screenshots:**
Workflow is working in docs repo, and it is set to cron jobs and manual as well.

[Screencast from 2024-07-31 00-07-33.webm](https://github.com/user-attachments/assets/d32d4f1c-7229-430e-a8c0-21246d1ffd23)

image

**Related Issue:**
- https://github.com/PalisadoesFoundation/talawa-api/issues/1897

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by gautam-divyanshu**
**What kind of change does this PR introduce?**

**Issue Number:**

Fixes #2426

**Snapshots/Videos:**

**If relevant, did you update the documentation?**

**Summary**

**Does this PR introduce a breaking change?**

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-api/blob/master/CONTRIBUTING.md)?**

## Summary by CodeRabbit

- **Chores**
	- Removed the `Generate-Documentation` job from the workflow, streamlining the process.
	- The `Empty-Commit` job has been updated to remove its dependency on the `Generate-Documentation` job.

### CODE REVIEW

**Comment by gautam-divyanshu** [comment]
@palisadoes @tasneemkoushar can we merge this pr?

File: .github/workflows/push.yml [hunk 1]
```diff
@@ -94,101 +93,6 @@ jobs:
...
-        #   persist-credentials: false
-      - name: Generate Documentation of HTML pages
-        run: |
-          npm install --global typedoc
-          npm install typedoc-plugin-markdown
-          npm i --save-dev @types/node
-          npx typedoc --entryPoints src --out talawa-api-docs --plugin typedoc-plugin-markdown --theme markdown --entryPointStrategy expand
-      
-      - name: Make Markdown Files MDX Compatible
-        run: python3 .github/workflows/md_mdx_format_adjuster.py --directory talawa-api-docs
```
Comment (palisadoes) [inline .github/workflows/push.yml:115]: Please remove the `./.github/workflows/md_mdx_format_adjuster.py`  script if it's not being used any more

**Review by palisadoes — CHANGES_REQUESTED** [review]
Please remove the `./.github/workflows/md_mdx_format_adjuster.py`  script if it's not being used any more

**Comment by gautam-divyanshu** [comment]
@palisadoes you can merge now

**Review by palisadoes — APPROVED** [review]

**Pull request merged**